chains/v4/chains_dev.json

```diff
@@ -3702,5 +3702,70 @@
         "color": "linear-gradient(315deg, #E04E02 0%, #FCB350 100%)",
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/DoraFactory.svg",
         "addressPrefix": 128
+    },
+    {
+        "chainId": "d611f22d291c5b7b69f1e105cca03352984c344c4421977efaa4cbdd1834e2aa",
+        "parentId": "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe",
+        "name": "Mangata X",
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "MGX",
```

Comment:
MGX should be done as ORML asset as well

Reply:
done